state-chain/pallets/cf-elections/src/lib.rs

```diff
@@ -1278,14 +1254,11 @@ pub mod pallet {
 				};
 
 				ensure!(
-					Self::with_electoral_access(
```

Comment:
The status check that this was previously doing is already done above in `ensure_can_vote` - so we only need to handle the corrupt storage here.